src/components/versions-table.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from "react";
+import { handleCopy } from "../utils/helpers";
+
+interface Compatibility {
+  // could be  (network: string) => string
+  network: string;
+  version: string;
+}
+
+interface Project {
+  title: string;
+  compatibilities: Compatibility[];
+  install: string;
+}
+
+interface CompatibilityData {
+  title: string;
+  projects: Project[];
+  networks: string[];
+}
+
+const GIST_URL =
+  "https://gist.githubusercontent.com/Ben-Rey/9f475f1c5e8e0f78fa1570a727eef344/raw/gistfile1.txt";
+
+export default function VersionsTable() {
+  const [data, setData] = useState<CompatibilityData>();
+
+  useEffect(() => {
+    fetch(GIST_URL)
+      .then((response) => response.json())
+      .then((data) => setData(data));
+  }, []);
+
+  const getInstall = (project: Project, network: string) => {
+    const compatibility: Compatibility = project.compatibilities.find(
+      (compatibility) => compatibility.network === network
+    );
+    return `${project.install}@${compatibility.version}`;
+  };
+
+  if (!data) {
+    return <div></div>;
+  }
+
+  return (
+    <div style={{ marginTop: "50px" }}>
+      <div>
```

Comment:
You should remove useless div:
```suggestion
  <>
```